app/controllers/TestTaskController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+class TasksController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /tasks
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+		$pTitle		=	"Tasks";
+
+		$user 		= 	User::find(Auth::id());		
+		$tasks 		= 	$user->tasks()->whereState("incomplete")->get();
+		$counter	=	0;
+
+		return View::make('tasks.index', compact(['tasks','counter','pTitle']));
+	}
+
+	/**
+	 * Show the form for creating a new resource.
+	 * GET /tasks/create
+	 *
+	 * @return Response
+	 */
+	public function create()
+	{	
+		// Rules
+		$rules	= array(
+				'weight' 	=> 'integer|between:1,5',
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "weight" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.109:9000/coding_rules#rule_key=php%3AS1192)